libs/service-portal/health/src/screens/Vaccinations/tables/SortedVaccinationsTable.tsx

```diff
@@ -0,0 +1,94 @@
+import { useLocale, useNamespaces } from '@island.is/localization'
+import { SortableTable, formatDate } from '@island.is/service-portal/core'
+import { messages } from '../../../lib/messages'
+import { Vaccine } from '../dataStructure'
+import { tagSelector } from '../../../utils/tagSelector'
+import { VaccinationsDetailTable } from './VaccinationsDetailTable'
+import { DetailHeader, DetailRow } from '../../../utils/types'
+import { ATC_URL_BASE } from '../../../utils/constants'
+
+interface Props {
+  data: Array<Vaccine>
+}
+export const SortedVaccinationsTable = ({ data }: Props) => {
+  useNamespaces('sp.health')
+  const { formatMessage } = useLocale()
+  const headerDataDetail: Array<DetailHeader> = [
+    {
+      value: 'Nr.',
+    },
+    {
+      value: 'Dags.',
+    },
+    {
+      value: 'Aldur',
+    },
+    {
+      value: 'Bóluefni',
+    },
+    {
+      value: 'Staður',
+    },
+  ]
+  return (
+    <SortableTable
+      title=""
+      labels={{
+        vaccine: formatMessage(messages.vaccinatedFor),
+        date: formatMessage(messages.vaccinatedLast),
+        status: formatMessage(messages.status),
+      }}
+      tagOutlined
+      expandable
+      sortByKey="vaccine"
```

Comment:
Is the user only able to sort on vaccine types?

Reply:
no this is the first rendering only, I'll change the name